### PR TITLE
[.github] add issue templates

### PR DESCRIPTION
**Pull request by pmli**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+**To Reproduce**
+Steps to reproduce the behavior.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```
Comment (renefritze) [inline .github/ISSUE_TEMPLATE/bug_report.md:22]: "Desktop" feels weird here. Maybe something like "software stack" could be better?

Comment (sdrave) [reply .github/ISSUE_TEMPLATE/bug_report.md:22]: I agree.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- OS: [e.g. iOS]
+- Python version
+- pyMOR version
```
Comment (sdrave) [inline .github/ISSUE_TEMPLATE/bug_report.md:25]: Maybe also NumPy/SciPy?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+Steps to reproduce the behavior.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+- OS: [e.g. iOS]
```
Comment (sdrave) [inline .github/ISSUE_TEMPLATE/bug_report.md:23]:
```suggestion
- OS: [e.g. macOS 12.0.1]
```

iOS is quite unlikely ..

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
```
Comment (sdrave) [inline .github/ISSUE_TEMPLATE/feature_request.md:17]: This sounds a bit like you need to first come up with ten alternatives before you are allowed to request a new feature. Maybe remove this point?

**Review by renefritze — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**System information (please complete the following information):**
+- Operating system: (`python -c 'import platform; print(platform.platform())'`)
+- Python version: (`python -V`)
+- pyMOR version: (`python -c 'import pymor; print(pymor.__version__)'`)
```
Comment (sdrave) [inline .github/ISSUE_TEMPLATE/bug_report.md:25]:
```suggestion
**System information:**
Output of `python -c 'import pymor; print(pymor.config)'`.
```

With #1483, both the Python version and the OS version are included in pymor.config.

**Review by sdrave — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/config.yml [hunk 6]
```diff
@@ -0,0 +1,4 @@
+contact_links:
+  - name: GitHub Discussions
+    url: https://github.com/pymor/pymor/discussions
+    about: For discussing other types of issues.
```
[the remark this answers is not in the document]
Comment (pmli) [reply .github/ISSUE_TEMPLATE/config.yml:4]: I also don't like it, so I was looking for other suggestions. I'll work on it some more.

**Review by sdrave — APPROVED** [review]

**Pull request merged**